js/RTCPeerConnection.js

```diff
@@ -584,6 +587,84 @@ RTCPeerConnection.prototype.createDataChannel = function (label, options) {
 	return new RTCDataChannel(this, label, options);
 };
 
+RTCPeerConnection.prototype.getStats = function () {
+	var self = this,
+		isPromise,
+		selector,
+		callback, errback;
+
+	if (typeof arguments[0] !== 'function') {
+		isPromise = true;
+		selector = arguments[0];
+	} else {
+		isPromise = false;
+		callback = arguments[0];
+		selector = arguments[1];
+		errback = arguments[2];
+	}
+
+	if (selector && !(selector instanceof MediaStreamTrack)) {
+		throw new Error('getStats() must be called with null or a valid MediaStreamTrack instance as argument');
+	}
+
+	if (isClosed.call(this)) {
+		return;
```

Comment:
This is new code, new code should be correct IMHO. If bugs are caught in the process they can be fixed later, but making the same mistake again is not right.